h/models/feature.py

```diff
@@ -20,6 +20,8 @@
     'client_display_names': "Render display names instead of user names in the client",
     'index_es6': ("Index annotations into Elasticsearch 6 "
                   "(only takes effect if enabled for everyone)?"),
+    'search_es6': ("Search annotations in Elasticsearch 6 "
+                   "(only takes effect if enabled for everyone)?"),
```

Comment:
The "only takes effect if enable for everyone" part is not true for the `search_es6` feature flag. The reason it applies for `index_es6` is because that feature flag is used in the context of a Celery worker process where there is never a logged-in user (ie. `celery.request.authenticated_user` is always `None` AFAIK).